Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -287,5 +287,14 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"CONNECTING" "Connecting..."
 		"INGAME_PLAYERS" "Players:"
 		"TOTAL_SERVERS" "Servers:"
+    
+    // Keybinds
+    "NORTHSTAR" "Northstar"
```

Comment:
Can you indent this to match the rest of the file (also convert it to tabs). I don't think Norsthstar needs to be localised.

Reply:
Should be good now